### PR TITLE
Some updates to keep page + group spelling

### PR DESCRIPTION
**Pull request by ailin-nemui**

### CODE REVIEW

**Comment by splitbrain** [comment]
I'm fine with 1b3c93d

The other change does not what you describe (it adjust the user not the group), also since AD is case-insensitive it makes sense to sanitize to lower case.

**Comment by ailin-nemui** [comment]
I see, we have been using the adfs connection on a generic SAML provider and our names are case-sensitive

**Comment by ailin-nemui** [comment]
would you accept it as a setting?

**Comment by splitbrain** [comment]
Yeah I think a setting makes sense.

**Comment by ailin-nemui** [comment]
updated

File: conf/metadata.php [hunk 1]
```diff
@@ -5,3 +5,4 @@
 
 $meta['endpoint']    = array('string');
 $meta['certificate'] = array('');
+$meta['casemapping'] = array('multichoice', '_choices' => array('lowercase', 'none'));
```
Comment (splitbrain) [inline conf/metadata.php]: this should be an ``onoff`` switch.

Comment (ailin-nemui) [inline conf/metadata.php]: ok

**Comment by splitbrain** [comment]
thx

**Pull request merged**